client/blocks/edit-gravatar/index.jsx

```diff
@@ -276,11 +276,8 @@ export class EditGravatar extends Component {
 					</p>
 					<InfoPopover className="edit-gravatar__pop-over" position="left">
```

Comment:
Minor thing but, can we add `text-align: justify` to the contents of this popup? I think it looks better:
![image](https://github.com/user-attachments/assets/716baa21-4d71-4e5b-8603-94ef15e26b2e)
vs
![image](https://github.com/user-attachments/assets/f1fd1774-1709-44a5-985d-f6944e91f6cf)

Reply:
I tried using `text-align: justify`, but the spacing between each word looks strange. Therefore, I ended up not adding the property.

Reply:
In the screenshot I made above I think it looks ok. But I guess it's a matter of opinion :). anyway, not a blocker!